isolate/test/index.ts

```diff
@@ -440,7 +440,7 @@ describe('isolate', function() {
       }
 
       function MyDataflowComponent(
-        sources: {other: any},
+        sources: {other: unknown},
```

Comment:
Even before this PR, this line should have been `unknown`.

But now, with this PR, this test does not pass (type check) with `any` (because `scopedSinks.other` would have been inferred as unknown, which would be a compile error a few lines below), so I had to change it to `unknown`.